build/Dockerfile.builder

```diff
@@ -3,19 +3,30 @@ FROM debian:buster as bazel-builder
 COPY build/install-bazel.sh /build/
 RUN chmod +x /build/install-bazel.sh && /build/install-bazel.sh
 
-# Build auth binary.
+# Copy in only the necessary files for building.
 FROM bazel-builder as auth-builder
-COPY . /src
+COPY Makefile WORKSPACE /src/
```

Comment:
Can we collapse these `COPY` statements into 1 to reduce the number of layers.

Reply:
I originally tried this in one statement like this:
`COPY Makefile WORKSPACE bazel config src test /src/`
but the problem is that if any of the things on the left are directories instead of copying the directory it copies the *contents* of the directory, so you end up with all the contents of your subdirectories directly in the root `src` directory. Do you know any good way to do this in one statement without this happening?

Reply:
Resolved this by using `.dockerignore` file to prevent the unnecessary files from being copied into the image.